File: curso-python/ex040.py
# crie um programa que leia duas notas de um aluno e calcule sua média, mostrando uma mensagem no final
# de acordo com a média atingida
# - média abaixo de 5.0:
    # - reprovado
# - média entre 5.0 e 6.9
    # - Recuperação
# - média 7.0 ou superior
    # - Aprovado

n1 = float(input("primeira nota: "))
n2 = float(input("segunda nota: "))
media = (n1+n2) / 2
print(f"você tirou as notas {n1} e {n2} e sua média foi {media:.2f}")

if media < 5.0:
    print("reprovado")
elif media >= 5.0 and media < 7.0:
    print("Recuperação")
else:
    print("aprovado")

File: curso-python/javascript-exercicios/ex040.js
const prompt = require("prompt-sync")()
let n1 = Number(prompt("primeira nota: "))
let n2 = Number(prompt("segunda nota: "))
let media = (n1+n2) / 2
console.log(`você tirou as notas ${n1} e ${n2} e sua média foi ${media.toFixed(2)}`)

if (media < 5){
    console.log("reprovado")
}else if ((media >= 5.0) && (media < 7.0)){
    console.log("recuperação")
}else{
    console.log("aprovado")
}